lib/rules/jsx-equals-spacing.js

```diff
@@ -61,7 +62,9 @@ module.exports = {
 
           const sourceCode = context.getSourceCode();
           const equalToken = sourceCode.getTokenAfter(attrNode.name);
+          // @ts-ignore
           const spacedBefore = sourceCode.isSpaceBetweenTokens(attrNode.name, equalToken);
+          // @ts-ignore
```

Comment:
ts-expect-error

Reply:
Thanks! Fixed in [eb44ba7](https://github.com/jsx-eslint/eslint-plugin-react/pull/3731/commits/eb44ba71aba5ed0d36518b3c658b7cab5e59c14f) .